src/main/java/core/basesyntax/dao/Storage.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    static final List<Fruit> FRUIT_LIST = new ArrayList<>();
```

Comment:
As for me, List not fits best for this task, I have used map and it saved much time